scripts/generate-locales.ts

```diff
@@ -0,0 +1,118 @@
+import ts from 'typescript';
+import path, { relative } from 'path';
+import { writeFile, mkdir, stat } from 'fs/promises';
+import { glob, chalk } from 'zx';
+
+const SRC_DIR = path.resolve('./src/lib/i18n/locales');
+const DEST_DIR = path.resolve('./static/i18n/locales');
+
+const logAndExit = (msg: string, logError: (msg: string) => void) => {
+  logError(chalk.red(msg));
+
+  if (logError === console.error) {
+    process.exit(0);
+  }
+};
+
+const relativeTo = (path: string) => {
```

Comment:
It seems a bit much, but I'm into it.

Reply:
Lol, I think I wrote it before I knew it was going to be a one liner.